src/main/resources/jupyter/init-actions.sh

```diff
@@ -140,7 +139,7 @@ if [[ "${ROLE}" == 'Master' ]]; then
       docker exec -u root -d ${JUPYTER_SERVER_NAME} ${JUPYTER_HOME}/${JUPYTER_USER_SCRIPT}
     fi
 
-    docker exec -d ${JUPYTER_SERVER_NAME} ${PYSPARK}
+    docker exec -d ${JUPYTER_SERVER_NAME} ${JUPYTER_NOTEBOOK}
```

Comment:
We shouldn't be running pyspark here anymore, we should be running `jupyter notebook` (to match the Dockerfile entrypoint). This must have been some earlier oversight.

Reply:
Should the auto-tests notice anything different from this change?

Reply:
Apparently not. It causes us to ALWAYS start a pyspark2 context, so python3+spark or R+spark would be running 2 spark context. Maybe there is enough Spark bandwidth so those tests still pass.

Reply:
ah thanks.  Yeah we should not always run a py2